### PR TITLE
Migrate reminderbot to rules

### PR DESCRIPTION
**Pull request by melindaloubser1**
**Proposed changes**:
- Move reminderbot to rules policy

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by melindaloubser1** [comment]
I meant to open this as draft, now I can't figure out how to change it 🤷‍♀️ Cancelling reminders & callback isn't working yet. I tried to test it using the chatroom front end, but I think I need to configure the callback URL on the chatroom side too?

**Comment by melindaloubser1** [comment]
Ok, so I still can't figure out how to test this from a front end, but it all seems to be working right based on logs.

File: examples/reminderbot/credentials.yml [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+# # NOTE: the rest channel does not work with external events
+# rest:
 # you don't need to provide anything here - this channel doesn't
 # require any credentials
 
 callback:
   # URL to which Rasa Open Source will send the bot responses
   # See https://rasa.com/docs/rasa/user-guide/connectors/your-own-website/#callbackinput
-  url: "http://localhost:5034/bot"
+  url: "http://localhost:5000/bot"
```
Comment (erohmensing) [inline examples/reminderbot/credentials.yml:13]:
```suggestion
  url: "http://localhost:5034/bot"
```

File: examples/reminderbot/callback_server.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        body = {"status": "message sent"}
+        return response.json(body, status=200)
+
+    return bot_app
+
+
+if __name__ == "__main__":
+    app = create_app()
+    port = 5000
```
Comment (erohmensing) [inline examples/reminderbot/callback_server.py:24]:
```suggestion
    port = 5034
```

File: examples/reminderbot/domain.yml [hunk 3]
```diff
@@ -1,38 +1,28 @@
...
 - action_forget_reminders
 - action_react_to_reminder
 - action_tell_id
 - action_warn_dry
-
-session_config:
-  session_expiration_time: 60  # value in minutes
-  carry_over_slots_to_new_session: true
+- utter_what_can_do
+- utter_goodbye
```
Comment (erohmensing) [inline examples/reminderbot/domain.yml:28]:
```suggestion
```

File: examples/reminderbot/credentials.yml [hunk 4]
```diff
@@ -20,11 +21,6 @@ callback:
 #  slack_token: "<your slack token>"
 #  slack_channel: "<the slack channel>"
 
-#socketio:
-#  user_message_evt: <event name for user message>
-#  bot_message_evt: <event name for bot messages>
-#  session_persistence: <true/false>
-
```
Comment (erohmensing) [inline examples/reminderbot/credentials.yml:27]: why was this removed?

Comment (melindaloubser1) [reply examples/reminderbot/credentials.yml:27]: arg sorry this was me trying to narrow down my issues

Comment (melindaloubser1) [reply examples/reminderbot/credentials.yml:27]: I uncommented them, then deleted them

File: examples/reminderbot/README.md [hunk 5]
```diff
@@ -24,14 +24,37 @@ To train and chat with `reminderbot`, execute the following steps:
...
+   ```
    
-3. Run a Rasa X to talk to your bot. 
-   If you don't have a Rasa X server running, you can test things with `rasa x` in a separate shell (the action server must keep running).
+   You can then send messages to the bot via the callback channel endpoint: 
+   ```
+   curl -XPOST http://localhost:5005/webhooks/callback/webhook \
+      -d '{"sender": "tester", "message": "hello"}' \
+      -H "Content-type: application/json"
+   ```
```
Comment (melindaloubser1) [inline examples/reminderbot/README.md:58]: Yes this worked great, thank you for writing the instructions & callback server!

I defined a function to make testing easier (I have the same for duckling et al) but I think this is shell specific so should not go in the instructions?

```
callback() {
d="{\"sender\": \"tester\", \"message\": \"$@\"}"
echo $d 
curl -XPOST http://localhost:5005/webhooks/callback/webhook \
      -d $d \
      -H "Content-type: application/json"
}
```

**Comment by erohmensing** [comment]
But also maybe lets wait to merge this if the format (`...` stuff) is going to change again 😅

File: examples/reminderbot/README.md (hunk 5 above)
Comment (erohmensing) [reply examples/reminderbot/README.md:58]: Ooh nice! I usually just test by sending messages via insomnia 😅

**Comment by melindaloubser1** [comment]
Sure, it can just live here for now =)

**Comment by akelad** [comment]
@erohmensing @melindaloubser1 i vote we maybe just merge this anyways? The `...` can be fixed with a second PR. I've ended up merging all the other example bots as well just because people need to see example of the format. 

But, the nlu file needs to be updated to include `-` in front of all the examples, since that changed now.

**Comment by akelad** [comment]
@erohmensing can you approve?
